### PR TITLE
:sparkles: Add Screenshot Test for Search Screen

### LINKED ISSUE #629: Add Screenshot Test for Search Screen
**Idea Description**
Implement a screenshot test to verify the functionality of search screen.
Just checking the launch of the app is above acceptant criteria. But you can implement more actions if you want.

**Reference images and links**
スクリーンショット 2023-08-17 20 32 58

### PR DESCRIPTION
**Pull request by Corvus400**
## Issue
- close #629 

## Overview (Required)
- Added ScreenShot test for SearchScreen.

### CODE REVIEW

**Comment by Corvus400** [comment]
I had originally implemented the test in #806, but after conflicts with others, I changed it to just add one test.

File: feature/sessions/src/test/java/io/github/droidkaigi/confsched2023/sessions/SearchScreenTest.kt [hunk 1]
```diff
@@ -79,6 +79,17 @@ class SearchScreenTest {
         }
     }
 
+    @Test
+    @Category(ScreenshotTests::class)
+    fun checkEmptyBodyShot() {
+        searchScreenRobot {
+            setupSearchScreenContent()
+            inputDummyTextSearchTextFieldAppBar()
+            checkExistEmptyBody()
```
Comment (takahirom) [inline feature/sessions/src/test/java/io/github/droidkaigi/confsched2023/sessions/SearchScreenTest.kt:88]: May I ask what we check here?
I thought we could spot change in the screen image easily. 👀

Comment (Corvus400) [reply feature/sessions/src/test/java/io/github/droidkaigi/confsched2023/sessions/SearchScreenTest.kt:88]: @takahirom 
This is a test to see if EmptyBody is displayed when a session is not found. 💁‍♂️

Comment (takahirom) [reply feature/sessions/src/test/java/io/github/droidkaigi/confsched2023/sessions/SearchScreenTest.kt:88]: I mean while checkScreenCapture() takes a screenshot of the screen, what checkExistEmptyBody() will check? I was thinking we could remove this 👀

**Review by takahirom — APPROVED** [review]
Thanks for covering the test!

**Pull request merged**